README.md

```diff
@@ -0,0 +1,34 @@
+# Documentation for the DHIL
+
+To build this documentation, first download Jekyll following the
+instructions here: https://jekyllrb.com/docs/installation/
+
+Mac OS comes with ruby already, but it may be prudent to have a higher version.
+In that case, you can `brew install ruby` and then add the following to your shell's
+profile:
+
+
+```bash
+## Ruby 
+if [ -d "/opt/homebrew/opt/ruby/bin" ]; then
+  export PATH=/opt/homebrew/opt/ruby/bin:$PATH
+  export PATH=`gem environment gemdir`/bin:$PATH
+fi
+```
```

Comment:
`brew` can be installed in a few different locations, so this might be a better way to make sure `ruby` is in the path.

``` shell
PFX=$(brew --prefix)
if [ -d "$PFX/opt/ruby/bin" ]; then
  export PATH=$PFX/opt/ruby/bin:$PATH
  export PATH=`gem environment gemdir`/bin:$PATH
fi
```